### ISSUE TITLE
Is it possible to evaluate vcf of tetraploid species?

### ISSUE THREAD

**Issue by afzm** [Illumina/hap.py#55]
I experienced this error:
[W] Found a variant with more 4 > 2 (max) alt alleles. These become no-calls.

**Comment by pkrusche** [comment]
At this point, hap.py is restricted to maximally two GT entries / diploid species. I started to add support for more general situations at some point here: https://github.com/Illumina/hap.py/blob/master/src/c%2B%2B/include/HapSetMatcher.hh ... but haven't continued working on this since it is not an application we have been actively looking into.

One issue with this is that the number of possible haplotype assignments increases when increasing the ploidy, which will increase compute cost to the point of becoming intractable. One thing to try could be allele-based comparisons if haplotype-based genotype matching is not essential to your application.

**Comment by afzm** [comment]
Yeah allele-based comparison can work, would it be easy to implement?
